sf-pcap-ng.c

```diff
@@ -856,7 +856,7 @@ pcap_ng_check_header(bpf_u_int32 magic, FILE *fp, u_int precision, char *errbuf,
 	 * If we find a bigger block, we reallocate the buffer.
 	 */
 	p->bufsize = 2048;
-	if (p->bufsize < total_length)
+	if ((bpf_u_int32) p->bufsize < total_length)
```

Comment:
This should no longer be necessary with 2d96be0322d71ce62e2f68d038070b10a7c9201b.